src/main/java/app/studnicki/ox/Round.java

```diff
@@ -0,0 +1,55 @@
+package app.studnicki.ox;
+
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+import java.util.HashMap;
+import java.util.Map;
+
+class Round {
+  final int dimension;
+  Map<Integer, Sign> board = new HashMap<>();
+
+  private final int limit;
+  private static final int MININUM_DIMENSION = 3;
```

Comment:
Should this be in Round or maybe settings?

Reply:
Delegated to Config class.